spikeinterface/core/testing_tools.py

```diff
@@ -9,6 +9,7 @@ def generate_recording(
         num_channels=2,
         sampling_frequency=30000.,  # in Hz
         durations=[10.325, 3.5],  #  in s for 2 segments
+        set_probe=True
```

Comment:
Can you set it to False to not break some previous stuff ?

Reply:
default was True

Reply:
I set it to False only when I don't need the probe

Reply:
OK